pages/calendar/index.tsx

```diff
@@ -9,12 +9,11 @@ export default function CalendarPage() {
   if (!data) return <div>Loading...</div>;
 
   const { events } = data;
-  const eventsList = events.sort((a, b) => a.start - b.start);
```

Comment:
This was indeed confusing candidates.

Although I think it should be moved [here](https://github.com/pacerevenue/frontend-interview/blob/main/utils/index.ts#L13) instead of the service. The events are sorted mostly for the greedy algorithm (it's a prerequisite). That was a bug in the logic before, and I think it was added before to the Calendar component just as a quick fix.

So I believe you can replace line 13 in that file with
```js
let events = eventsList.toSorted((a, b) => a.start - b.start);
```